documentation/book/proc-accessing-resource-status.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='proc-accessing-resource-status-{context}']
+= Checking the status of a custom resource
+
+This procedure describes how to find the status of a custom resource.
+
+.Prerequisites
+
+* An {OpenShiftName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Specify the custom resource in a standard JSONPath expression to filter on the `status` property:
```

Comment:
```suggestion
. Specify the custom resource and use `-o jsonpath` option to apply a standard JSONPath expression to select the `status` property:
```

Reply:
Use a single bullet point for a single-step procedure.

Reply:
Perhaps's Tom's edited text could be split into unordered sub-steps?

Reply:
> Use a single bullet point for a single-step procedure.

My personal feeling on this is that people will identify a step with a number and it's good to have that consistency for recognition. But I understand if it's the convention to use a bullet for < 2 steps.

Reply:
> Perhaps's Tom's edited text could be split into unordered sub-steps?

It's a single command, so I guess we leave as a single step.